modules/ROOT/pages/using-api-alerts.adoc

```diff
@@ -11,7 +11,7 @@ An API alert is an alarm that flags one of the following problems:
 * The API returns a specified HTTP error code.
 * The API response time exceeds a certain amount.
 
-Runtime Manager alerts differ in nature from API alerts. Runtime Manager flags CPU usage, secure data gateway up/down, and deployment success/failure, to name a few. API alerts, on the other hand, relate to an API request or response.
+xref:runtime-manager::alerts-on-runtime-manager.adoc[Runtime Manager alerts] differ in nature from API alerts. Runtime Manager flags CPU usage, secure data gateway up/down, and deployment success/failure, to name a few. API alerts, on the other hand, relate to an API request or response.
```

Comment:
The topic you're linking to is called "Alerts", so maybe move Runtime Manager out of the link?

Reply:
Done! Thanks!